CONTRIBUTING.adoc

```diff
@@ -62,9 +60,7 @@ bug workarounds.
 
 === How to build the code with profiling
 
-VIf you launch `nix-shell` with the `enableHaskellProfiling` argument set to true, you will get a shell where all the dependencies have been built with profiling.
-
-Like this: `nix-shell --arg enableHaskellProfiling true`.
+VIf you launch `nix develop .#prof`, you will get a shell where all the dependencies have been built with profiling.
```

Comment:
Typo 
```suggestion
If you launch `nix develop .#prof`, you will get a shell where all the dependencies have been built with profiling.
```

Reply:
Heh I assumed this was some formatting thing.